app/src/main/java/org/piwigo/bg/AlbumService.java

```diff
@@ -49,20 +50,18 @@ protected void onHandleIntent(Intent intent) {
             @Override
             public void onResponse(Call<AddCategoryResponse> call, Response<AddCategoryResponse> response) {
                 if (response.raw().code() == 200) {
-                    if (response.body().stat.equals("ok")) {
-                        // TODO: make text localizable
-                        Toast.makeText(getApplicationContext(), "Created album: " + catName + " !", Toast.LENGTH_LONG).show();
-                    }
+                    if (response.body().stat.equals("ok"))
```

Comment:
The two nested if blocks have exactly the same code, did you consider combining already the conditions and keep only one if(response.raw().code() == 200 && response.body().stat.equals("ok"))?